src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Ball {
+
+    private int number;
+    private String color;
+
+    public void setNumber(int number) {
+        this.number = number;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public String toString() {
+
+        StringBuilder builder = new StringBuilder();
+        builder.append(color).append(number);
+        return builder.toString();
+    }
```

Comment:
```suggestion
    @Override
    public String toString() {

        StringBuilder builder = new StringBuilder();
        builder.append(color).append(number);
        return builder.toString();
    }
```

Нам нету смысла использовать тут StringBuilder, потому что у нас нету конкатенации строк в цикле, давай уберем его

Reply:
Agree, removed using StringBuilder